src/Operators/products_between_fields_and_grid_metrics.jl

```diff
@@ -1,14 +1,14 @@
 #####
-##### Operators of the form A * q where A is an area and q is some quantity.
+##### Operators of the form grid_metric * q, where grid_metric is a grid metric and q is some variable
```

Comment:
```suggestion
##### Operators of the form grid_metric * q
```

Since "some variable" isn't anymore specific than simply writing "q"